Proposals/20200511-HVPA-design.md

```diff
@@ -191,6 +191,43 @@ New Requests: 225m / 2.7Gi with 2 replicas, because 1.8Gi lands us in the “sec
 Notice: Old and new CPU would even land us in bucket 1, but memory brings us only down to bucket 2, so we go to bucket 2 with 2 replicas.
 Reasoning: We don’t know what the cache percentage is and if it is very low, the remaining replicas will need even more memory to cope with the requests. If not, VPA will eventually scale down.
 
+#### Providing stability at bucket boundaries (TBD):
+Given that the **buckets are decided based on total resource allocation, and NOT the actual replica sizes**, if the total memory recommendation in the above example fluctuates around 2GB, then HVPA controller would be switching repeatedly between 1st and 2nd bucket.
```

Comment:
Can you please rephrase to clarify the general statement without referring to the example? The example can be a follow up statement to explain?

Reply:
done in https://github.com/gardener/hvpa-controller/pull/71/commits/33aef9bd412492ab2b43623ef7cdd8fce0f65e8d.